### ISSUE TITLE
[Feature Request]: On Spigot, have unknown blocks in CustomObjects/Structures be placed as air instead of noteblocks

### ISSUE THREAD

**Issue by ZeddGrayhem** [PG85/OpenTerrainGenerator#960]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Suggested Feature

For Gehenna, I've been working on these trees, and they include amethyst crystals, but those don't exist till 1.17 and above.

We still want to support Paper 1.16.5, but we don't want to have to split the code into two branches. Paper replaces unknown blocks with air automatically. However, Spigot uses note blocks instead, which won't look good.

It would look fine if it were replaced with air and not noteblocks though.

### Suggested Implementation

Have a method in the Spigot platform code that reviews any blocks that come in from bo3s/bo4s, and checks if they exist.

If they do, return what's inputted.

If not, return SpigotMaterials.AIR

### Anything else?
![image](https://user-images.githubusercontent.com/93233032/173663146-2d47747f-1e34-4a15-80d9-14f93d6d2085.png)

**Comment by authvin** [comment]
Yes, I think a toggle might be nice, defaulting to air. I will try to get this done this weekend.

**Comment by StupidCornfield-Auto** [comment]
Ok so we fixed the problem.

Apparently the note block code was literally placed there by PG85 in June 2021.

The fact that otg had thought for over a year that spigot puts noteblocks in place of unrecognized blocks but the whole time otg was doing it and pg85 made a test method and never changed it is about as funny as Syphilis
